test/RateTargetBaseInterestRateStrategy.t.sol

```diff
@@ -57,12 +57,28 @@ contract RateTargetBaseInterestRateStrategyTest is InterestRateStrategyBaseTest
         assertEq(interestStrategy.getBaseVariableBorrowRate(), 0.055e27);
         assertEq(interestStrategy.getMaxVariableBorrowRate(),  0.515e27);
 
-        rateSource.setRate(0.07e27);
+        rateSource.setRate(0.07e18);
+        rateSource.setDecimals(18);
 
         assertEq(interestStrategy.getVariableRateSlope1(),     0.01e27);
         assertEq(interestStrategy.getVariableRateSlope2(),     0.45e27);
         assertEq(interestStrategy.getBaseVariableBorrowRate(), 0.075e27);
```

Comment:
Would be good to add a little comment here saying that it's still 27 precision with 18 decimal rate source

Reply:
fixed